src/lib/zendesk/realZendesk/realZendeskService.ts

```diff
@@ -55,12 +55,16 @@ export default class RealZendeskService implements ZendeskInterface {
         await instance
             .post("/api/v2/tickets.json", data)
             .then(function (response) {
-                console.log(response);
+                console.log("Zendesk service response status: ", response.status);
+                console.log("Zendesk service response status text: ", response.statusText);
                 sent = true;
             })
             .catch(function (response) {
                 sent = false;
-                console.log(response);
+                console.log("Zendesk service response code: ", response.code);
+                console.log("Zendesk service response status: ", response.response.status);
+                console.log("Zendesk service response status text: ", response.response.statusText);
+                console.log("Zendesk service response data: ", response.response.data);
```

Comment:
What's likely to be in the zendesk response data here?
We'll want to ensure we don't have PII going to places it shouldn't be? Like some of our log sources?

Reply:
@huwd  there is no PII in those. Those are to help identify potential error. If you have anymore questions, please ask in the Jira for the story as this is a public repository.